frontend/src/metabase/static-viz/containers/StaticChart/StaticChart.tsx

```diff
@@ -26,6 +27,21 @@ import { FUNNEL_CHART_TYPE } from "../../components/FunnelChart/constants";
 import { StaticChartProps } from "./types";
 
 const StaticChart = ({ type, options }: StaticChartProps) => {
+  const hasData = !!(
```

Comment:
Can you destructure it first to simplify the subsequent logic? e.g. `const { data, series } = options`